### ISSUE TITLE
Specifying numeric environment variables results in errors

### ISSUE THREAD

**Issue by allen-ziegenfus** [liferay/liferay-cli#112]
I am setting up a client extension as follows: 

```
public-site-navigation:
    name: Public Site Navigation
    scopes:
        - liferay-json-web-services.everything.write
        - liferay-json-web-services.everything
    type: oauthApplicationHeadlessServer
    userAccountEmailAddress: test@$[conf:dxp.lxc.liferay.com.virtualInstanceId]
runtime:
    cpu: 2000m
    env:
        - name: OAUTH2_JOB_PROFILE
          value: public-site-navigation
        - name: GROUP_ID
          value: "20121"
    memory: 500Mi
    type: configuration
    workload: job
```

However this fails on deploy with this error: 

```
STEP 4/4 — Deploying
     Running cmd: /repo/scripts/k8s/delete.py 2>/dev/null && /repo/scripts/k8s/create.py
     Error from server (BadRequest): error when creating "/tmp/create-669957da-7910-11ed-86ac-803f5d0dfcff.yaml": Job in version "v1" cannot be handled as a Job: json: cannot unmarshal number into Go struct field EnvVar.spec.template.spec.containers.env.value of type string
```

I've tried every combination I can think of for specifying the value of "GROUP_ID": with or without quotes, specifying !!string and !!integer and trying something I found on some kubernetes issue threads:  `{{ 20121 | quote }}` and nothing seems to work. Is it simply not possible to pass any numbers as environment variable values?

**Comment by gamerson** [comment]
hey @allen-ziegenfus  I reproduced this and submitted a fix.

**Comment by allen-ziegenfus** [comment]
awesome thanks @gamerson  :rocket:

**Comment by gamerson** [comment]
Fixed in liferay/liferay-localdev#105
